src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemove.stories.tsx

```diff
@@ -64,8 +64,8 @@ export const AutomaticRemoveViewStory: Story = {
             href: "/user/dashboard/fix/high-risk-data-breaches/social-security-number",
           }}
           currentSection="data-broker-profiles"
-          monthlySubscriptionUrl=""
-          yearlySubscriptionUrl=""
+          monthlySubscriptionUrl="http://mozilla.com/subscribe/monthly"
+          yearlySubscriptionUrl="http://mozilla.com/subscribe/yearly"
```

Comment:
Just to ensure it's clear to everyone that these are mocks (I was confused and visited the URLs :sweat_smile:):

```suggestion
          monthlySubscriptionUrl="https://example.com/subscribe/monthly"
          yearlySubscriptionUrl="https://example.com/subscribe/yearly"
```